FamousSaying/Home/CardView/CardView.swift

```diff
@@ -30,41 +39,74 @@ final class CardView: UIView {
         makeConstraints()
+        setDelegate()
     }
     
     required init?(coder: NSCoder) {
         fatalError()
     }
     
     private func makeSubviews() {
-        [quoteLabel,
+        [favoriteButton,
+         quoteLabel,
          animeLabel,
          characterLabel].forEach { self.addSubview($0) }
     }
     
     private func makeConstraints() {
-        let quoteLabelConstraints = [quoteLabel.topAnchor.constraint(equalTo: self.topAnchor, constant: 16),
+        let favoriteButtonConstrains = [favoriteButton.topAnchor.constraint(equalTo: self.topAnchor, constant: 20),
+                                        favoriteButton.trailingAnchor.constraint(equalTo: self.trailingAnchor, constant: -20)]
+        
+        let quoteLabelConstraints = [quoteLabel.centerYAnchor.constraint(equalTo: self.centerYAnchor),
                                      quoteLabel.leadingAnchor.constraint(equalTo: self.leadingAnchor, constant: 20),
                                      quoteLabel.trailingAnchor.constraint(equalTo: self.trailingAnchor, constant: -20)]
         
-        let characterLabelConstraints = [characterLabel.topAnchor.constraint(equalTo: quoteLabel.bottomAnchor, constant: 12),
+        let characterLabelConstraints = [characterLabel.bottomAnchor.constraint(equalTo: animeLabel.topAnchor, constant: -4),
                                          characterLabel.leadingAnchor.constraint(greaterThanOrEqualTo: self.leadingAnchor),
                                          characterLabel.trailingAnchor.constraint(equalTo: self.trailingAnchor, constant: -20)]
         
-        let animeLabelConstraints = [animeLabel.topAnchor.constraint(equalTo: characterLabel.bottomAnchor),
-                                     animeLabel.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: -16),
+        let animeLabelConstraints = [animeLabel.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: -16),
                                      animeLabel.leadingAnchor.constraint(greaterThanOrEqualTo: self.leadingAnchor),
                                      animeLabel.trailingAnchor.constraint(equalTo: characterLabel.trailingAnchor)]
         
         
-        [quoteLabelConstraints,
+        [favoriteButtonConstrains,
+         quoteLabelConstraints,
          characterLabelConstraints,
          animeLabelConstraints].forEach { NSLayoutConstraint.activate($0) }
     }
+    
+    func setDelegate() {
+        let leftSwipeGesture = UISwipeGestureRecognizer(target: self, action: #selector(didLeftSwipe))
+        leftSwipeGesture.direction = .left
+        
+        let rightSwipeGesture = UISwipeGestureRecognizer(target: self, action: #selector(didRightSwipe))
+        rightSwipeGesture.direction = .right
+        
+        [leftSwipeGesture,
+         rightSwipeGesture].forEach { self.addGestureRecognizer($0) }
+        
+        favoriteButton.addTarget(self, action: #selector(didTapFavoriteButton), for: .touchUpInside)
```

Comment:
```suggestion
        favoriteButton.addTarget(delegate, action: #selector(delegate.didTapFavoriteButton), for: .touchUpInside)
```

target을 delegate로 바꾸고, delegate의 method를 직접 호출하는것도 가능하지 않,,,을까요,, ?(확인은 못해봄)

Reply:
해당 delegate 프로토콜에다 @objc를 추가해서 해보니 잘 되네요!!! 신기합니다아..

Reply:
바로 위에 있는 swipe 제스처도 똑같이 적용을 하니 잘 되네요..!

Reply:
반영 사항: https://github.com/pagh2322/FamousSays/commit/4ae1ea6a5736983a8ad8d2bc1c254460ca9ceebe